security/pod-to-pod-encryption.md

```diff
@@ -1,14 +1,11 @@
 ---
-title: Try out the pod-to-pod encryption tech preview
-description: Try out enabling WireGuard for state-of-the-art cryptographic security between pods for Calico clusters.
+title: Pod to pod encryption
+description: Enable WireGuard for state-of-the-art cryptographic security between pods for Calico clusters.
 ---
 
 ### Big picture
 
-Enable WireGuard to secure on the wire pod-to-pod traffic in a {{site.prodname}} cluster.
-
-> **Warning!** Pod-to-pod encryption is a tech preview and should not be used in production clusters. It has had very limited testing and it will contain bugs (please report these on the Calico Users Slack or GitHub). This feature is currently not supported with overlay networks (IP in IP or VXLAN) due to known issues with NodePort services.
-{: .alert .alert-danger}
+Enable WireGuard to secure on the wire pod-to-pod pod traffic in a {{site.prodname}} cluster.
```

Comment:
Could re-phrase this too if we're going for encrypt in-cluster pod traffic.  i.e. replace "pod-to-pod" with "in-cluster pod"